### PR TITLE
Moved Secretshare operator to quay.io/opencloudio

### PR DESCRIPTION
**Pull request by mikekaczmarski**

### CODE REVIEW

File: build/resources/extra/secretshare_deploy.yaml [hunk 1]
```diff
@@ -26,7 +25,6 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: secretshare
-  namespace: ibm-common-services
```
Comment (horis233) [inline build/resources/extra/secretshare_deploy.yaml:29]: @mikekaczmarski Why do we need to remove the namespace here

Comment (chenzhiwei) [reply build/resources/extra/secretshare_deploy.yaml:29]: maybe a rebase conflict?

Comment (mikekaczmarski) [reply build/resources/extra/secretshare_deploy.yaml:29]: Yes, looks like an oversight.  I will re-do

**Pull request merged**